Simplified.xcodeproj/project.pbxproj

```diff
@@ -6081,6 +6044,63 @@
 			defaultConfigurationName = Release;
 		};
 /* End XCConfigurationList section */
+
+/* Begin XCRemoteSwiftPackageReference section */
+		73579044276BE7F2009F1ADF /* XCRemoteSwiftPackageReference "iOS-Utilities" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/NYPL-Simplified/iOS-Utilities";
+			requirement = {
+				branch = "task/IOS-294/utilities";
```

Comment:
Do we need to update this before merging? Same for the Overdrive framework reference below

Reply:
oh yeah, definitely... this PR is not ready yet, but I wanted to get a head start to see if I was gonna get build errors... and of course I did get some 🤦‍♂️

Reply:
Oh sure, let me know when it's ready.

Reply:
this is now ready, although I will still need to update the commit refs once I merge the dependent PRs.